enos/ansible/roles/common/tasks/main.yml

```diff
@@ -17,3 +17,15 @@
 
 - name: Mount /run
   command: mount --make-shared /run
+
+- name: Ensure nscd is not loaded
+  shell: systemctl list-units --all nscd.service |fgrep "0 loaded units listed"
```

Comment:
Ansible service module cannot be used ?

Reply:
It change nothing. Same problem, `service` and `systemd` module failed if the service is not loaded.

Reply:
One idea could be to use the service module and ignore the error (if any).

https://docs.ansible.com/ansible/latest/playbooks_error_handling.html#ignoring-failed-commands

Reply:
done